src/components/popper/PopperComponent.js

```diff
@@ -0,0 +1,45 @@
+// @flow
+import * as React from 'react';
+import { Manager, Reference, Popper } from 'react-popper';
+import type { Modifiers, Placement } from 'popper.js';
+import { PLACEMENT_AUTO } from './constants';
+
+type Props = {
+    children: React.Node,
+    isOpen?: boolean,
+    modifiers?: Modifiers,
+    placement?: Placement,
+};
+
+const PopperComponent = (props: Props) => {
+    const { children, isOpen, modifiers, placement: popperPlacement = PLACEMENT_AUTO } = props;
```

Comment:
Definitely test a popper-ed component within something like a react-virtualized table  row that sets overflow hidden. The pop-up may get cut off. In that case you need to pass in an additional prop to this component like isPortaled. And if true, you portal the content.

Reply:
For reference see https://github.com/FezVrasta/react-popper/issues/79 and https://github.com/FezVrasta/popper.js/issues/567

Reply:
Can this be added in a small follow up PR so I can merge this larger PR?